shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearch.ts

```diff
@@ -67,11 +81,33 @@ export class JudgeActivityReportSearch extends JoiValidationEntity {
     judgeName: joi
       .optional()
       .description('The last name of the judge to generate the report for'),
-    startDate: JoiValidationConstants.ISO_DATE.max('now')
-      .required()
+    judges: joi
+      .array()
+      .items(joi.string())
+      .description('The last names of judges to generate report for'),
+    pageNumber: joi
+      .number()
+      .optional()
+      .description(
+        'The page of the selected subset of CAV and Submitted cases for display',
+      ),
+    pageSize: joi
+      .number()
+      .optional()
       .description(
-        'The start date to search by, which cannot be greater than the current date, and is required when there is an end date provided',
+        'The page size for calculating the number of CAV and Submitted cases to display.',
       ),
+    searchAfter: joi
+      .number()
+      .description(
+        'The last docket number to be used to make the next set of calls per page number',
+      ),
+    startDate: JoiValidationConstants.ISO_DATE.max('now').description(
```

Comment:
Why is start date not required anymore?

Reply:
`[JudgeActivityReportSearch.ts](https://github.com/ustaxcourt/ef-cms/pull/3799/files/96a9b0554976f8cb2c91b92a04349e86bfdda5bd#diff-5495bccafdb9396bb5d4653ecd9f9af032d074bc54f330583627f215fb754b2b) is used for the validation for the tables on this page and not all of those calls need to use the date parameters.

Reply:
But a start date and end date are required to run the report, right? So they must be present in the request in order to consider the report search valid?